tests/ui/lint/wasm_c_abi_transition.stderr

```diff
@@ -0,0 +1,59 @@
+error: this function function definition is affected by the wasm ABI transition: it passes an argument of non-scalar type `MyType`
+  --> $DIR/wasm_c_abi_transition.rs:17:1
+   |
+LL | pub extern "C" fn my_fun(_x: MyType) {}
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
```

Comment:
It's not technically correct that this will become a hard error (we'll just do the ABI switch), but I don't know how to make an FCW that does not have this message...